internal/controller/controllers/hypershiftagentserviceconfig_controller.go

```diff
@@ -118,6 +130,34 @@ func (hr *HypershiftAgentServiceConfigReconciler) getKubeconfigSecret(ctx contex
 	return secret, nil
 }
 
+func (hr *HypershiftAgentServiceConfigReconciler) ensureSpokeAgentInstallCRDs(ctx context.Context) error {
+	// Fetch agent-install CRDs using in-cluster client
+	localCRDs := &apiextensionsv1.CustomResourceDefinitionList{}
+	listOpts := []client.ListOption{
+		client.MatchingLabels{fmt.Sprintf("operators.coreos.com/assisted-service-operator.%s", hr.Namespace): ""},
```

Comment:
CRDs aren't namespaced, right? Why would this need the be the namespace the operator is running in?

Reply:
Right, but the label includes the namespace (i.e. operators.coreos.com/assisted-service-operator.assisted-installer: ""). Using it as an indication for finding the relevant CRDs, didn't find a cleaner way:/

Reply:
But is that actually the namespace the operator is installed in?
I'm not sure why the CRD would have a label related to where the operator is running.

That said, is there some way to query all crds for a particular api group/version?
For example, anything that has `spec: group: "agent-install.openshift.io"` in the CRD?